alacritty_windows.yml

```diff
@@ -276,12 +276,12 @@ live_config_reload: true
 
 # Shell
 #
-# You can set `shell.program` to the path of your favorite shell, e.g. `/bin/fish`.
+# You can set `shell.program` to the path of your favorite shell, e.g. `cmd`.
 # Entries in `shell.args` are passed unmodified as arguments to the shell.
-shell:
-  program: cmd
-  #args:
-  #  - --login
+#shell:
+#  program: powershell
+#  args:
+#    -
```

Comment:
Uncommenting this `shell` block should work properly without any issues. However the `args:` section could cause issues here.

I'd recommend just changing `args` to an empty array like this: `args: []`.

Reply:
It might be a tiny bit harder for folks to actually specify args if they don't know yaml, but I made the change.

Reply:
Or supply an example use case. Just a thought.

Reply:
I had supplied an example use case in a previous commit because I also thought that was a reasonable idea, but specifying the default is reasonable as well.